docs/Support/glossary.md

```diff
@@ -6,19 +6,16 @@ sidebar_position: 2
-4. **[Lit Actions](/coreConcepts/LitActionsAndPKPs/litActions#what-are-lit-actions)** — Immutable Javascript functions stored on IPFS that can use PKPs for programmatic signing and decryption. They can be thought of as Lit’s native version of smart contracts.
-5. **Distributed Key Generation** — a cryptographic process in which key generation requires participation from above a certain threshold of nodes, where each node only holds a ******share****** of the complete keypair.
-6. **[Encryption](/SDK/Explanation/encryption)** — the process of encoding data so that it remains hidden or inaccessible to unauthorized parties, the core technology that enables privacy on the Internet. At a high level, encryption converts plaintext (i.e “this is a secret”) into “nonsense” ciphertext (”e5sVVb#bn332J”). You can read a more in-depth explanation [here](https://www.cloudflare.com/learning/ssl/what-is-encryption/).
-7. **[Decentralized Access Control](/coreConcepts/accessControl/intro)** — Lit’s decentralized access control protocol allows you to encrypt information behind [on-chain conditions](https://developer.litprotocol.com/coreConcepts/accessControl/intro), allowing you to securely store data on the open web.
-8. **[Proof](/coreConcepts/LitActionsAndPKPs/litActions#proofs)** — A proof is a particular application for a digitial signature. For example, using a signature to *prove* that a particular interaction took place.
-    
-    Signing through Lit Actions opens up the possibilities of verifying information from external sources, such as from a Weather API. Within the Lit ecosystem this signed information is called a proof.
-    
-9. **[Signature](/coreConcepts/LitActionsAndPKPs/litActions#signing)** — In cryptography, a digital signature proves that a user controls the private key associated with a particular public address. This is commonly used in the context of *authorization*. Apps can use this signature to verify a user has control over the digital identity provided by the address, and provide authentication to do both web3 and arbitrary, non-web3 actions. For example, changing a profile on OpenSea requires signing in order to confirm the association between the profile owner the the wallet associated with it.
-    
-    In working with the Lit network, someone will sign with their wallet when they want to mint a PKP. That is a transaction signature — where someone is sending a transaction (minting). This signature is then used as the method of authentication over the PKP and associated Lit Actions.
\ No newline at end of file
+### **[Decentralized Access Control](/coreConcepts/accessControl/intro)** — Lit’s decentralized access control protocol allows you to encrypt information behind [on-chain conditions](https://developer.litprotocol.com/coreConcepts/accessControl/intro), allowing you to securely store data on the open web.
```

Comment:
Let's put the description on a new line, since having it combined with the title also puts the text in the sidebar

![Screen Shot 2022-11-30 at 4 13 33 PM](https://user-images.githubusercontent.com/5404516/204935304-ee8e796d-5e04-4258-96fd-16a301c3103e.png)